forch/authenticator.py

```diff
@@ -148,7 +151,7 @@ def parse_args(raw_args):
                         help='RADIUS server ip')
     parser.add_argument('-p', '--server-port', type=int, default=1812,
                         help='Server port that freeradius is listening on')
-    parser.add_argument('-r', '--radius-secret', type=str, default='SECRET',
+    parser.add_argument('-r', '--radius-secret', type=str, default='echo SECRET',
```

Comment:
somethings wrong, "echo SECRET" is not the same as the radius_secret itself. I think in this case (command line arg), it should be the secret, and the default value would be SECRET (without echo). No need to have the helper for a command line argument.

Reply:
But the command line arguments essentially builds the config data structure. This was why I had left the 'secret' field in place previously. Authenticator is fed this information through the config file. We either bring back the secret field or change the name of the command line argument,

Reply:
Updated. PTAL.